### ISSUE TITLE
MDC Slideable Drawers broken in RTL (with solution).

### ISSUE THREAD

**Issue by theenoahmason** [material-components/material-components-web#2048]
### What MDC Web Version are you using?
0.28

### What browser(s) is this bug affecting?

All. This is a css issue.

### What OS are you using?

All. this is a css issue.

### What are the steps to reproduce the bug?

Go to your slideable drawer demos, both `temporary` and `persistent`.
Add `dir="rtl"` to `<html>` element. 

The problem is instantly apparent. Toggle the drawer to investigate further.

### What is the expected behavior?

In RTL, drawer should start off screen and slide onto the screen.

### What is the actual behavior?

in RTL, drawer starts on screen and slides the wrong way, never having left the viewport.

#### This is easily solved following the same pattern you are currently using:

in LTR, you are using:
```
.mdc-drawer__drawer {
    transform: translateX(calc(-100% - 20px));
}
```

in RTL, you should be using the opposite:
```
*[dir="rtl"] .mdc-drawer__drawer {
    transform: translateX(calc(100% + 20px));
}
```

The reason for this is that you are using `transform`, but positioning the drawer initially with `right: 0;` in RTL.

**Comment by bonniezhou** [comment]
Duplicate of #1930, which should be fixed by #1957!

**Comment by theenoahmason** [comment]
Awesome, looks good.
